README.md

```diff
@@ -5,6 +5,7 @@
 ## Resources
 
 - [Production site](https://www.google.com.au/)
+- [Documentation](https://github.com/irene2mana-T3A2-2023/SakuraPantryDocs)
```

Comment:
I don't think we need this part here, since we are now in the documentation repository.